front/src/applications/stdcm/utils/__tests__/simulationReportSheet.spec.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  generateCodeNumber,
+  formatCreationDate,
+  getStopDurationTime,
+  extractSpeedLimit,
+  getOperationalPointsWithTimes,
+} from 'applications/stdcm/utils/formatSimulationReportSheet';
+
+import { suggestedOperationalPoints, simulation, departure_time } from './sampleData';
+
+describe('Utils simulation report sheet functions', () => {
+  test('should return a formatted string', () => {
+    const codeNumber = generateCodeNumber();
+    expect(codeNumber).toMatch(/^\d{2}\d{2}-\d{3}-\d{3}$/);
+  });
+
+  test('should format the date correctly', () => {
+    const formattedDate = formatCreationDate(departure_time);
```

Comment:
Could you update the TODO of this function to `TODO DROP STDCM V1` to help the process when we will do this task

Reply:
done